### PR TITLE
[FIX] Could not find or load main class org.apache.zookeeper.ZooKeeperMain

### LINKED ISSUE #10029: Could not find or load main class org.apache.zookeeper.ZooKeeperMain
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

3.0.0   sh ./bin/install.sh  

3.stop server
17.0.237.209:default
17.0.133.220:default
17.0.133.220:default
17.0.206.7 master server is stopping
Begin stop master-server......
no master-server to stop
End stop master-server.
17.0.137.109 master server is stopping
Begin stop master-server......
no master-server to stop
End stop master-server.
17.0.237.209 worker server is stopping
Begin stop worker-server......
no worker-server to stop
End stop worker-server.
17.0.133.220 worker server is stopping
Begin stop worker-server......
no worker-server to stop
End stop worker-server.
Begin stop alert-server......
no alert-server to stop
End stop alert-server.
17.0.237.209 api server is stopping
Begin stop api-server......
no api-server to stop
End stop api-server.
4.delete zk node
Error: Could not find or load main class org.apache.zookeeper.ZooKeeperMain

### What you expected to happen

Cluster installation

### How to reproduce

Cluster installation

### Anything else

no

### Version

3.0.0-alpha

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Desperado2**
## Purpose of the pull request

fix: #10029 

## Brief change log

update remove-zk-node.sh

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: script/remove-zk-node.sh [hunk 1]
```diff
@@ -37,11 +37,12 @@ source ${BIN_DIR}/env/dolphinscheduler_env.sh
...
-export DOLPHINSCHEDULER_LIB_JARS=$DOLPHINSCHEDULER_HOME/lib/*
+export DOLPHINSCHEDULER_LIB_JARS=$DOLPHINSCHEDULER_HOME/libs/*
 
 export DOLPHINSCHEDULER_OPTS="-Xmx1g -Xms1g -Xss512k -XX:+DisableExplicitGC -XX:+UseConcMarkSweepGC -XX:+CMSParallelRemarkEnabled -XX:LargePageSizeInBytes=128m -XX:+UseFastAccessorMethods -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=70 "
 export STOP_TIMEOUT=5
 
+registryServers=$REGISTRY_ZOOKEEPER_CONNECT_STRING
 CLASS=org.apache.zookeeper.ZooKeeperMain
 
 exec_command="$DOLPHINSCHEDULER_OPTS -classpath $DOLPHINSCHEDULER_CONF_DIR:$DOLPHINSCHEDULER_LIB_JARS $CLASS -server $registryServers rmr $rootNode"
```
Comment (SbloodyS) [inline script/remove-zk-node.sh]: We should replace `$registryServers` to `$REGISTRY_ZOOKEEPER_CONNECT_STRING`. So that we can define one less variable.

**Comment by SbloodyS** [comment]
I think this is a serious issue of pseudo-cluster installation. So I put this to `3.0.0-beta-2` milestone. WDYT? @devosend

File: script/remove-zk-node.sh (hunk 1 above)
Comment (Desperado2) [reply script/remove-zk-node.sh]: This is right. I have modified the file. thanks

File: script/remove-zk-node.sh [hunk 2]
```diff
@@ -37,14 +37,14 @@ source ${BIN_DIR}/env/dolphinscheduler_env.sh
 export JAVA_HOME=$JAVA_HOME
 
 export DOLPHINSCHEDULER_CONF_DIR=$DOLPHINSCHEDULER_HOME/conf
-export DOLPHINSCHEDULER_LIB_JARS=$DOLPHINSCHEDULER_HOME/lib/*
+export DOLPHINSCHEDULER_LIB_JARS=$DOLPHINSCHEDULER_HOME/api-server/libs/*
```
Comment (SbloodyS) [inline script/remove-zk-node.sh:40]: I'm not qute sure if there is a better way to achive this. PTAL @kezhenxu94

**Review by SbloodyS — APPROVED** [review]
LGTM. Thanks.

**Comment by SbloodyS** [comment]
Thanks for your contribution. This is a good beginning. Considering that it's your first contribution, I think we can get deep communication, you can contact me through mail or add WeChat(SbloodyS), when mail or added, please tell me who you are. I think I can help to familiar with the Dolphinscheduler if you meet with problems. @Desperado2

**Comment by devosend** [comment]
> I think this is a serious issue of pseudo-cluster installation. So I put this to `3.0.0-beta-2` milestone. WDYT? @devosend

Agrees with that. I will cherry-pick later.

**Pull request merged**